Consul/Consul.csproj

```diff
@@ -15,36 +13,24 @@
     <Authors>PlayFab,G-Research</Authors>
     <Description>Consul.NET is a .NET client library for the Consul HTTP API</Description>
     <PackageOutputPath>../dist/consul</PackageOutputPath>
-    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
-    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
-    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
-    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
-    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <PublicSign>true</PublicSign>
     <AssemblyOriginatorKeyFile>../assets/consuldotnet.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
   </PropertyGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+  <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
     <PackageReference Include="ILMerge" Version="3.0.41" PrivateAssets="All" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" PrivateAssets="All" />
+    <PackageReference Update="Newtonsoft.Json" PrivateAssets="All" />
```

Comment:
why the subtle change from `Include` to `Update` here? In case of `net461` TFM, it is not straightforward what happens as both lines will run:
```
<PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
<PackageReference Update="Newtonsoft.Json" PrivateAssets="All" />
```

Reply:
Hmm, I cannot find good docs for it now 😞 But basically `Update` updates already defined `PackageReference`.  So in our case it is going to be equivalent to:
`<PackageReference Include="Newtonsoft.Json" Version="10.0.3" PrivateAssets="All" />`

My goal was to avoid specifying the version twice.

Reply:
> My goal was to avoid specifying the version twice.

I see. In the context of a multi targeting library I find this approach less explicit though. Another solution could be:

```
<NewtonsoftVersion>10.0.3</NewtonsoftVersion>

<PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftVersion)" />
<PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftVersion)" PrivateAssets="All" />
```

No strong feelings though

Reply:
One more benefit of "update" approach is that I was able to get rid of the condition in line 32. With your proposal it would still need to be there.

btw, the docs are here: https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2019#attributes-and-elements

Reply:
Not convinced, but no strong feelings :-)